flask_restplus/api.py

```diff
@@ -65,6 +65,7 @@ class Api(object):
     :param str contact: A contact email for the API (used in Swagger documentation)
     :param str license: The license associated to the API (used in Swagger documentation)
     :param str license_url: The license page URL (used in Swagger documentation)
+    :param str host: The host that the api is running on.  Not always the localhost, thought it is the default.
```

Comment:
I am still a bit confused by this.  I have added: 

```
from werkzeug.contrib.fixers import ProxyFix
app.wsgi_app = ProxyFix(app.wsgi_app)
```
to my code and it does not return a host in the swagger doc.
{
basePath: "/v1",
consumes: [
"application/json"
],
info: {
description: "API  Interface",
title: "Tudor Instrument API",
version: "1.0.0"
},
paths: {},
produces: [],
responses: {},
swagger: "2.0",
tags: []
}


When setting SERVER_NAME explicitly it causes errors. (404)

Could you elaborate on how that should work.

Also,  I want to re-confirm the use case.

We are generating a swagger document.  Storing in a mongo db.  Calling it from another location for dynamic function generation.

It looks to me like if I am setting SERVER_NAME it impacts the routing.  I was hoping for a solution where the app behavior isn't driven by the doc.

That said, I don't have a huge amount of experience in this area. Could you provide a bit more clarification on:
  
1) how the proxy fix should behave
2) whether I can have a completely arbitrary host name